debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlConnectorIT.java

```diff
@@ -2618,6 +2621,47 @@ public void shouldEmitTruncateOperation() throws Exception {
 
         stopConnector();
     }
+    
+    @Test
+    public void taskStartShouldNotWaitOnSchemaHistoryRecovery () throws InterruptedException {
+        // Introduce a delay of 10 seconds in recovering every record in schema history. This is 
+        // done to increase the time taken to recovery schema history.
+        Configuration config = DATABASE.defaultConfig()
+            .with(SCHEMA_HISTORY_RECOVERY_DELAY_MS, 10_000)
+            .build();
+
+        start(MySqlConnector.class, config);
+        logger.info("Sleeping for 10 seconds to allow connector to start and commit an offset");
```

Comment:
Q: How is the connector able to commit offset in 10 seconds?

If it is not, the history recovery code-path won't be executed and the test will pass irrespectively.